schedview/compute/maf_metrics.py

```diff
@@ -1,12 +1,13 @@
 from copy import deepcopy
 
 from rubin_sim import maf
-import schedview
-import schedview.compute.astro
+from rubin_sim.maf import MetricBundleGroup  # noqa F401
 
 # Imported to help sphinx make the link
 from rubin_sim.scheduler.model_observatory import ModelObservatory  # noqa F401
-from rubin_sim.maf import MetricBundleGroup  # noqa F401
+
+import schedview
+import schedview.compute.astro
```

Comment:
I think this has been bothering me (and maybe other things in this same part of schedview) because I don't know if I understand the use case. I can see that "things are calculated with MAF" here, but in general, wouldn't you want to pre-calculate the MAF metrics and then only show the outputs here, so that any "calculation of things with MAF" should actually happen with a batch and script that calls MAF directly? Rather than running it via schedview? 

And if you had a bundle that you wanted to start execution of with MAF, wouldn't the better approach be to set up a bundle and script within MAF, and then execute that script (in rubin_sim) with a trigger from schedview?

Reply:
If I recall correctly, I wrote this for use in notebooks, without dashboards in mind. In any case, this file isn't used anywhere in the dashboards, and I'm not using those notebooks anymore, so the file can just be removed, I'll do that.

Reply:
I do think there will need to be a home for utilities that compute stuff with maf that are needed for specific plots needed on rare occasions (or only for parameters that are bespoke to a few situations), but not generally applicable enough to be computed routinely or automatically. I don't have an opinion on whether this code should be part of maf or schedview, though.